internal/controller/volumereplicationgroup_controller.go

```diff
@@ -694,6 +701,54 @@ func (v *VRGInstance) updatePVCList() error {
 	return v.separatePVCsUsingStorageClassProvisioner(pvcList)
 }
 
+func (v *VRGInstance) labelPVCsForCG() error {
```

Comment:
I am just mentioning a conversation we had in another PR here:
https://github.com/RamenDR/ramen/pull/1472#discussion_r1689995421

This works for now. We will just have one CG per DRPC.